frontend/src/pages/Authentication/SignInPage.js

```diff
@@ -21,14 +22,16 @@ function SignInForm(props) {
   // otherwise breaks the 'Rules of Hooks' apparently.
   // ref: https://stackoverflow.com/questions/60700905/react-native-navigate-to-screen-invalid-hook-call
   const navigate = useNavigate();
+  const { saveToken } = useAuth()
 
   // Called when the user clicks the submit button. Observe how
   // the handleSubmit button is attached to the 'form' component
   // below in the return call of SignUpForm function.
   const handleSubmit = event => {
     event.preventDefault();
-
-    let data = { formInput };
+    
+    console.log(formInput)
```

Comment:
Is this necessary? If not should be removed.

Reply:
It is not necessary. I forgot it there after testing.